lib/db.js

```diff
@@ -91,6 +106,34 @@ DB.prototype.getTransactionWithBlockInfo = function(txid, queryMempool, callback
   });
 };
 
+DB.prototype.sendTransaction = function(tx, callback) {
+  if(tx instanceof this.Transaction) {
+    tx = tx.toString();
+  }
+  $.checkArgument(typeof tx === 'string', 'Argument must be a hex string or Transaction');
+
+  try {
+    var txid = this.bitcoind.sendTransaction(tx);
+    return callback(null, txid);
+  } catch(err) {
+    return callback(err);
+  }
+};
+
+DB.prototype.estimateFee = function(blocks, callback) {
+  var self = this;
+
+  // For some reason getting fee for 1 block returns -1
+  // Until this is resolved, just make it 2 blocks
```

Comment:
Some information on this:
> The estimated fee the transaction should pay in order to be included within the specified number of blocks. If the node doesn’t have enough information to make an estimate, the value -1 will be returned
https://bitcoin.org/en/developer-reference#estimatefee